rollup.config.js

```diff
@@ -5,6 +5,7 @@ import { nodeResolve } from '@rollup/plugin-node-resolve';
 import postcss from 'rollup-plugin-postcss';
 import stringHash from 'string-hash';
 import cssUrl from 'postcss-url';
+import walkSync from 'walk-sync';
```

Comment:
I have a few concerns about us adding an extra dependency here. We should in theory just be able to use `fs.readdirsync` from node to do all of this. In fact... there are kind of examples of how this would work in our Docs, as we are using just basic node functions to crawl our repo directories and find the MDX files. Is there a strong reason we are adding another dependency here instead of just using node?

Reply:
makes sense, working on it.

Reply:
Dependency removed.